[chore]: #4231 Update MAINTAINERS (dev branch) (#4233)
hyperledger/iroha: MAINTAINERS.md

## Maintainers

> Maintainers of Hyperledger Iroha project

Reviewer: ```suggestion
Maintainers of the Hyperledger Iroha project
```

are supposed to help contributors by explaining them project details,

> such as architecture, process, existing issues.

Reviewer: ```suggestion
such as architecture, process, and existing issues.
```

> This is the list of maintainers, including their email address for direct communications:

Reviewer: ```suggestion
This is the list of maintainers, including their email addresses for direct communication:
```

|         Name          |                      GitHub Id                       |              email              |                               Area of expertise                               |
| --------------------- | ---------------------------------------------------- | ------------------------------- | ----------------------------------------------------------------------------- |
| Makoto Takemiya       | [@takemiyamakoto](https://github.com/takemiyamakoto) | takemiya@soramitsu.co.jp        | Product vision                                                                |
| Dmitri Venger         | [@dmitrivenger](https://github.com/dmitrivenger)     | venger@soramitsu.co.jp          | Project manager                                                               |
| Bogdan Mingela        | [@Mingela](https://github.com/Mingela)               | mingela@soramitsu.co.jp         | Iroha Team Lead                                                               |
| Aleksandr Petrosyan   | [@appetrosyan](https://github.com/appetrosyan)       | ap886@cantab.ac.uk              | Iroha 2 architect                                                             |

> | Daniil Polyakov       | [@Arjentix](https://github.com/Arjentix)             | daniil.polyakov@soramitsu.co.jp | Development: Rust                                                             |

Reviewer 2: ```suggestion
| Daniil Polyakov       | [@Arjentix](https://github.com/Arjentix)             | daniil.polyakov@soramitsu.co.jp | Development: Rust, C++                                                             |
```

As I can see we're adding additional languages here

| Grzegorz Bazior       | [@baziorek](https://github.com/baziorek)             | bazior@agh.edu.pl               | Development: C++, Python, JavaScript, TypeScript                              |
| Andrey Kostiuchenko   | [@arndey](https://github.com/arndey)                 | kostiuchenko@soramitsu.co.jp    | Developer: Java, Kotlin                                                       |
| Timur Guskov          | [@gv-timur](https://github.com/gv-timur)             | guskov@soramitsu.co.jp          | Development: Java                                                             |
| Dmitriy Creed         | [@Cre-eD](https://github.com/Cre-eD)                 | creed@soramitsu.co.jp           | DevSecOps                                                                     |
| Vasily Zyabkin        | [@BAStos525](https://github.com/BAStos525)           | zyabkin@soramitsu.co.jp         | DevOps                                                                        |
| Ekaterina Mekhnetsova | [@outoftardis](https://github.com/outoftardis)       | mekhnetsova@soramitsu.co.jp     | Documentation                                                                 |

> | William Richter       | [@WRRicht3r](https://github.com/WRRicht3r)           | ryan.rrr66@gmail.com            | Documentation                                                                 |

Reviewer: ```suggestion
| William Richter       | [@WRRicht3r](https://github.com/WRRicht3r)           | richter@soramitsu.co.jp            | Documentation                                                                 |
```
